app/src/main/res/values/colors.xml

```diff
@@ -29,4 +29,15 @@
     <color name="maroon">#B04040</color>
     <color name="maroon_shadow">#A0A0A0</color>
     <color name="colorPrimaryLight">#DFDFDF</color>
+
+<!-- FROM DESIGN LIBRARY AT https://www.figma.com/file/0qkRp3TrKBRnpEnmQeA5m4/-->
```

Comment:
Honestly good call, I like this a little more than my arbitrary "colorPrimary/Secondary/etc" designations

Reply:
Thanks!  Glad you like it :)